src/services/snapshots.actions.ts

```diff
@@ -736,34 +881,76 @@ export function updateV4GroupUrls(snapshot: NormalizedSnapshot): void {
   }
 }
 
+export async function getStoredSnapshots() {
+  let stored
+  try {
+    stored = await browser.storage.local.get<Stored>('snapshots')
+  } catch (err) {
+    return Logs.err('Snapshots.vue: recalcSizes: Cannot get snapshots', err)
+  }
+  return stored.snapshots
+}
+
+export async function prepareExport(snapshot: Snapshot | SnapshotState) {
+  // const parsed = parseSnapshot([snapshot], 0)
+  // console.log({parsed});
+
+  // if(!parsed)return
+  const { id, time, containers, sidebar, tabs } = snapshot //parsed
+  const normSnapshot = { id, time, containers, sidebar, tabs }
+
+  console.log({ normSnapshot, snapshot })
+
+  const jsonStr = JSON.stringify(normSnapshot)
+  const jsonFile = new Blob([jsonStr], { type: 'application/json' })
+  const mdStr = convertToMarkdown(normSnapshot as NormalizedSnapshot)
+  const mdFile = new Blob([mdStr], { type: 'text/markdown' })
+  return { id, time, containers, sidebar, tabs, jsonFile, mdFile }
+}
+
+export async function getMostRecentSnapshot() {
```

Comment:
getMostRecentSnapshot, getMostRecentSnapshotMd are not used, remove?

Reply:
i thought i would need them, and still might if we want to decouple timing of snapshot creation and export.
i can remove them for now.

Reply:
https://github.com/gotjoshua/sidebery/commit/9cd02a6aa9c2e0a40c228914b9e0c106f8d0f4e9